### DISCUSSION TITLE
Sensor 1 orientation

### DISCUSSION THREAD

**Discussion by Surreal62** [srozum/film_camera_tester#51] [Q&A]
Something I had meant to post some weeks ago.
In the description of Sensor 1, it states: "Such construction is commonly used in professional testers, also allows to use it without worrying about the sensor orientation."
However, I have found that it appears orientation does matter.
As an example, tester set to FP Vert Time, with a Pentax ME Super in Manual set to 1/60th sec, first test produces To: 5.77ms. Tc: 5.87ms.
Invert the sensor and the result is To: 5.87ms Tc: 5.77ms.
Same results when in Fp Vert Curt.
When tester set to FP Horiz Time, with an OM-3Ti set to 1/60 sec, first test produces To: 12.55ms. Tc: 12.66ms. 
Inverting the sensor produces To: 12.66ms and Tc: 12.55ms.

Does anyone else using the unit find the same?

**Comment by srozum** [comment]
Hi, if you see different readings in different sensor orientations, check your assembly. You either have something in sensor openings or they not aligned properly.

**Reply by Surreal62** [reply]
Nope. Spotlessly clean holes and perfectly aligned photocells. Assembly is as accurate as it can be. 
I have used the original sensor housing as designed by you and also the updated sensor housing face by the contributor User-462 (which, incidentally, when SLA printed automatically aligns the photo cells perfectly and prints the holes at exactly 0.8mm) and both behave in the same manner.

**Reply by srozum** [reply]
Well, what else I can say. It should measure the same in both directions.  There’s nothing in software to cause the difference. 
If you don’t see any issues with assembly quality it may be internal differences in photo sensors. You can try to solder another one if you have spares.

**Reply by Surreal62** [reply]
I will do that for sure. I have plenty of spares. 
It is something I noticed only by chance when I was setting the tester up originally,
prior to making a sensor holder and I realised I used the sensor inverted from what I had always done previously.
If it does happen there is this oddity on occasion, it is worth knowing about.

Don't get me wrong - your tester really is a brilliant piece of equipment - every bit as good as any Arrowin/Kyoritsu (and I checked mine directly against one and within a very narrow margin of a difference of equipment types, both produced the same results!) and serves me very, very well indeed! 👍👍👍👍

One thing that would really be above and beyond though - if there is space/connectivity available on the Arduino, would there be any chance the test results could be output to some form of memory card?
I know the Nano can do such - and there is another tester available than can do - but it isn't nearly as advanced as yours.
